Student-Unity-frontend/src/app/components/sing-up/recaptcha-api.js

```diff
@@ -0,0 +1,31 @@
+/* PLEASE DO NOT COPY AND PASTE THIS CODE. */ (function () {
```

Comment:
Why it's typed here ``/* PLEASE DO NOT COPY AND PASTE THIS CODE. */``

Reply:
It's from google documentation that you have to use the official way by including the library, although this is way neat but unfortunately it's not supported for localhost anymore. So I must include the script by my self to render to captcha.